Sources/Imaging/Hybrid/SampleFunction/index.js

```diff
@@ -0,0 +1,118 @@
+
+import macro     from 'vtk.js/Sources/macro';
+import vtkDataArray from 'vtk.js/Sources/Common/Core/DataArray';
+import vtkImageData from 'vtk.js/Sources/Common/DataModel/ImageData';
+
+const { vtkErrorMacro } = macro;
+
+// ----------------------------------------------------------------------------
+// vtkSampleFunction methods
+// ----------------------------------------------------------------------------
+
+function vtkSampleFunction(publicAPI, model) {
+  // Set our className
+  model.classHierarchy.push('vtkSampleFunction');
+
+  // Capture "parentClass" api for internal use
+  const superClass = Object.assign({}, publicAPI);
+
+  publicAPI.getMTime = () => {
+    if (model.implicitFunction === undefined) {
```

Comment:
`if (!model.implicitFunction)` will be more generic as it will handle `null` and `undefined`.

And better could be

`if (!(model.implicitFunction && model.implicitFunction.getMTime))`